### ISSUE TITLE
General error: PDO::FETCH_LAZY

### ISSUE THREAD

**Issue by eva-thien** [phalcon/cphalcon#10353]
HI all

I have the code below

```
            $pdo = $this->getDI()->getDb();
            $query = $pdo->prepare($sql);
            $query->setFetchMode(\Phalcon\Db::FETCH_ASSOC);
            $query->execute();
            $result = $query->fetchAll();
````
It working in Phalcon version 1.3.x, but After I have update Phalcon version 2.0 then I get errors

```
PDOException: SQLSTATE[HY000]: General error: PDO::FETCH_LAZY can\'t be used with PDOStatement::fetchAll()
```

**Comment by andresgutierrez** [comment]
Can you try again compiling from the 2.0.x branch?

**Comment by andresgutierrez** [comment]
This seems to be a missing feature of PDO https://bugs.php.net/bug.php?id=44541

**Comment by andresgutierrez** [comment]
This is fixed in 2.0.x branch
